test/statistic_test.cc

```diff
@@ -289,16 +305,39 @@ TEST(StatisticTest, HdrStatisticPercentilesProto) {
   EXPECT_EQ(json, golden_json);
 }
 
+TEST(StatisticTest, CircllhistStatisticPercentilesProto) {
```

Comment:
Would it be hard to duplicate the code here with `StatisticTest.HdrStatisticPercentilesProto`?

Reply:
If we were to have a shared utility function e.g. `loadGoldFile(parsed_json_proto, gold_file_path)`, it would only dedup 2 lines of code which seems not providing much benefit. I slightly prefer current form since it makes the test setup logic locally in the test case (easier to navigate).

Reply:
makes sense, sounds good to me then.